products/kdeplasma.md

```diff
@@ -17,12 +17,12 @@ auto:
 
 releases:
 -   releaseCycle: "5.26"
-    latest: "5.26.5"
+    latest: "5.26.90"
```

Comment:
Looks odd.

Reply:
Looked into this, the KDE tagging scheme is somewhat confusing. I think `.90` tags are used for beta releases. See https://download.kde.org/unstable/plasma/ vs https://download.kde.org/stable/plasma/ for eg.

The [info page](https://kde.org/info/plasma-5.26.90/) doesn't say anything, but the [announcement page](https://kde.org/announcements/plasma/5/5.26.90/) just calls it 5.27 beta.

@usta any ideas? We could add a regex to avoid `.9x` patch releases easily. Looking at [past releases](https://github.com/endoflife-date/release-data/blob/main/releases/kdeplasma.json), it only seems to be `.95` and `.90`.

Would be nice if this is documented somewhere.